jdat_notebooks/NIRISS_WFSS_postpipeline/00. Optimal extraction.ipynb

```diff
@@ -4,18 +4,28 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## 00 Optimal extraction of NIRISS WFSS spectrum;\n",
+    "## 00 Optimal Extraction of NIRISS WFSS Spectrum\n",
     "\n",
-    "This notebook aims to do 1D optimal extraction, whereas the JWST pipeline only provides a box extraction, which improves S/N of spectra for faint sources.\n",
+    "**Use case:** optimal extraction of grism spectra; redshift measurement; emission-line maps.  Simplified version of [JDox Science Use Case # 33](https://jwst-docs.stsci.edu/near-infrared-imager-and-slitless-spectrograph/niriss-example-science-programs/niriss-wfss-with-nircam-parallel-imaging-of-galaxies-in-lensing-clusters).<br>\n",
+    "**Data:** JWST simulated NIRISS images from [MIRAGE](https://jwst-docs.stsci.edu/jwst-other-tools/mirage-data-simulator), run through the [JWST calibration pipeline](https://jwst-pipeline.readthedocs.io/en/latest/); galaxy cluster.<br>\n",
+    "**Tools:**  specutils, astropy, pandas, emcee, lmfit, corner, h5py.<br>\n",
+    "**Cross-intrument:** NIRSpec <br>\n",
+    "**Documentation:** This notebook is part of a STScI's larger [post-pipeline Data Analysis Tools Ecosystem](https://jwst-docs.stsci.edu/jwst-post-pipeline-data-analysis).<br>\n",
     "\n",
-    "This notebook will start with post-pipeline products of NIRISS WFSS, 2D rectified spectra, from spec level3.\n",
     "\n",
+    "## Introduction\n",
+    "\n",
+    "This notebook is 1 of 4 in a set focusing on NIRISS WFSS data:\n",
+    "    1. 1D optimal extraction since the JWST pipeline only provides a box extraction.  Optimal extraction improves S/N of spectra for faint sources.\n",
+    "    2. Combine and normalize 1D spectra.\n",
+    "    3. Cross correlate galaxy with template to get redshift.\n",
+    "    4. Spatially resolved emission line map.\n",
     "\n",
-    "- Optimal extraction requires source morphology along the cross-dispersion direction, where we will retrieve from direct images taken along with WFSS observations.\n",
+    "This notebook will start with post-pipeline products of NIRISS WFSS, 2D rectified spectra, from spec level3.\n",
```

Comment:
Is there a clear and succinct description what  the various JWST pipeline levels mean somewhere in jdox?  Might be good to link that here if so.

Reply:
Link added to pipeline documentation.  Forgot to respond to this comment though.